deployment_report/model/utils/ingress_util.py

```diff
@@ -103,3 +109,65 @@ def ignorable_for_controller_if_unavailable(ingress_controller: Text, resource_t
 
     # not ignorable
     return False
+
+
+def get_version(kubectl: KubectlInterface) -> Optional[Text]:
```

Comment:
`get_version` might be a bit vague since there are so many possible versions that it could be getting. How about `get_ingress_version` for more clarity and since other methods include "ingress" in the name.